solidity/contracts/bridge/BridgeState.sol

```diff
@@ -88,7 +88,9 @@ library BridgeState {
         // the moving funds process. Moving funds wallets having their BTC
         // balance below that value can begin closing immediately as
         // transferring such a low value may not be possible due to
-        // BTC network fees.
+        // BTC network fees. The value of this parameter must always be smaller
+        // than `redemptionDustThreshold` in order to prevent redemption requests
+        // with values smaller or equal to `movingFundsDustThreshold`.
         uint64 movingFundsDustThreshold;
```

Comment:
```suggestion
        // BTC network fees. The value of this parameter must always be lower
        // than `redemptionDustThreshold` in order to prevent redemption requests
        // with values lower or equal to `movingFundsDustThreshold`.
        uint64 movingFundsDustThreshold;
```

Reply:
Done in https://github.com/keep-network/tbtc-v2/pull/241/commits/f2a5d262f26294ddcbdaf4295aa33b3cc18a5418.